docs/pages/admin-guides/deploy-a-cluster/deployments/aws-ha-autoscale-cluster-terraform.mdx

```diff
@@ -12,48 +12,61 @@ and describe how to manage the resulting Teleport deployment.
 
 ## Prerequisites
 
-Our code requires Terraform 0.13+. You can [download Terraform here](https://www.terraform.io/downloads.html). We will assume that you have
-`terraform` installed and available on your path.
+- Terraform 0.13+. You can [download Terraform
+  here](https://www.terraform.io/downloads.html). We will assume that you have
```

Comment:
We should update this link to https://developer.hashicorp.com/terraform/install.

Also, Terraform 0.13 is pretty old, you can't download a version older than that without trying really hard. Maybe we don't need to mention this anymore?

Reply:
Changed in 567bd0d9c33f9ab148ef63ac187709b781f798f3